frontend/src/pages/admin/UserManagement.jsx

```
@@ -0,0 +1,357 @@
+    console.log('Xóa người dùng với ID:', userId)
+
+    setDeleteModal({ open: true, id: userId })
+  }
+
+  // Filtered data based on searchText
+  const filteredData = userData.filter((user) => {
+    const search = searchText.trim().toLowerCase()
+    if (!search) return true
+    return (
+      (user.name && user.name.toLowerCase().includes(search)) ||
+      (user.email && user.email.toLowerCase().includes(search)) ||
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)